radio/src/targets/horus/board.cpp

```diff
@@ -57,16 +57,16 @@ bool UNEXPECTED_SHUTDOWN()
   if (WAS_RESET_BY_WATCHDOG())
     return true;
   else if (WAS_RESET_BY_SOFTWARE())
-    return RTC->BKP0R != SOFTRESET_REQUEST;
+    return getRTCBKPR(RTCBKP0R) != SOFTRESET_REQUEST;
   else
-    return RTC->BKP1R == POWER_REASON_SIGNATURE && RTC->BKP0R != SHUTDOWN_REQUEST;
+    return getRTCBKPR(RTCBKP1R) == POWER_REASON_SIGNATURE && getRTCBKPR(RTCBKP0R) != SHUTDOWN_REQUEST;
```

Comment:
I would prefer we define this code somewhere shared between the different targets, as it seems to be used in the same way at different places. To me this seems to be part of the software rather than some board specific thing.

Reply:
UNEXPECTED_SHUTDOWN() and SET_POWER_REASON() are identical for horus and nv14 but not for taranis and sigmspace. I think it is target specific with two targets having the same requirement.

Reply:
Do they need to be different for taranis based targets and horus/NV14 based targets?

Reply:
I believe we should be able to use the same code.